### ISSUE TITLE
Routed Email (Email alias) returns 

### ISSUE THREAD

**Issue by hydrian** [ScottPeterJohnson/purelymail-issues#177]
Here is how it is reproduced with username and domain sanitized. 
Create mail user: user@mydomain
Created routed alias:  

Domain: mydomain.com | Exact Match: full.name | Send To: user@mydomain.com

I can send an email to user@mydomain.com fine. If I send an e-mail to user@mydomain.com it is received fine. If I send an e-mail to full.name@mydomain.com I get the following error returned to the e-mail sender:

We could not deliver the attached mail for the following recipients.
full.name@mydomain.com (Rerouted): User isn't located here (5.1.1)

**Comment by hydrian** [comment]
Switching to 'Address starting with' doesn't change anything.

**Comment by ScottPeterJohnson** [comment]
As explained on Discord and for future reference via search:
TLDR: This is symbolic subaddressing in action. Just turn it off for mydomain.com (under the Routing page).
Slightly longer: By default domains have symbolic subaddressing enabled, which translates full.name to fullname by default, so full.name as a routing rule won't match anything.

**Comment by hydrian** [comment]
While this was a user TL;DR issue, I think the UI could have expressed it better.

- Have checkbox setting above the mail routing rules as it effects how rules are written.
- In the description of the setting, explicitly say what characters are considered symbol characters. Most places only use the '+' symbol from what I've experienced. So full.name+spam@mydomain.com would resolved to full.name@mydomain.com. If you are worried about a wall of text in the routing setting page, keep the list of characters behind a tool-tip or new tab/window.
